mkdocs.yml

```
@@ -85,6 +85,7 @@ nav:
     - user-guide/app_deletion.md
     - user-guide/best_practices.md
     - user-guide/status-badge.md
+    - user-guide/argocd-cli.md
```

```suggestion
    - user-guide/commands/argocd-cli.md
```

I hope I'm not missing anything